workshops/Tkinter/README.md

```diff
@@ -0,0 +1,135 @@
+---
+name:'Creating a UI with Tkinter.'
+description:'We will use python tkinter graphics to make User Interfaces. Its a nice world.'
+author: '@Bezlin Johnson'
+---
+
+# Final User Interface or UI.
+
+<img width="539" alt="Screenshot 2020-09-19 at 11 28 35 AM" src="https://user-images.githubusercontent.com/54895485/93660382-e47c0880-fa6b-11ea-8f9a-94f476991652.png">
+
+This will be the simple UI we will be making today. This is nice world we can even make cool softwares with this .
+Here is the [live_demo](https://repl.it/@bezlin/graphics#main.py) and [final_code](https://repl.it/@bezlin/graphics#main.py) (see src/app.js and public/app.css files).
+This workshop will take 50 minutes.
+![ezgif com-video-to-gif](https://user-images.githubusercontent.com/54895485/93660884-07f58200-fa71-11ea-8bbd-44b10c7624b5.gif)
```

Comment:
This intro is a bit dry, try to hype up the project they'll be making

Reply:
I don't know standard English but I will try

Reply:
ok cool